src/template-parameters.json

```diff
@@ -264,32 +264,13 @@
         "name": "cookieDuration",
         "help": "The cookie duration in milliseconds (default to `15552000000`, 6 months).",
         "type": "TEXT",
-        "simpleValueType": true,
-        "valueValidators": [
-          {
-            "type": "POSITIVE_NUMBER"
-          }
-        ]
+        "simpleValueType": true
       },
       {
         "displayName": "User Token",
         "name": "initialUserToken",
         "help": "Set an initial user identifier applied to subsequent events.",
         "type": "TEXT",
-        "simpleValueType": true,
-        "valueValidators": [
-          {
-            "args": ["^.{1,64}$"],
-            "type": "REGEX"
-          }
```

Comment:
I'm skipping all the validations within GTM. All the events will reach Algolia Insights API. If it fails, it will be noticeable on [Events Debugger](https://algolia.com/events/debugger). Plus, we don't know when we will change this validation rules.